docs/howto/getting-started/enable-openstack-cli.md

```diff
@@ -0,0 +1,171 @@
+you typed the password correctly, you must re-type it below. This 
+password should be adequately strong, and thus a password manager may 
+come in handy.
+
+![Set username and password](assets/ostack-cli/shot-03.png)
+
+Scroll down a bit, so the _Regions_ section is in full view. Expand one 
+or more of the available regions you want your new user to have access 
+to. For each one of the expanded regions, select one or more 
+_Projects_. For each project, activate one or more _Roles_. (Hint: For
+an overview of the rights that roles provide, hover the mouse pointer
+over the exclamation mark icon by the _Roles_.) 
+
+![Regions, projects, and roles](assets/ostack-cli/shot-04.png)
+
+Optionally, type in a description for the new OpenStack user. Then, 
+create the user by clicking the green _Create_ button below the 
+_Description_ box.
+
+![User description and creation](assets/ostack-cli/shot-05.png)
+
+The new OpenStack user will be ready in just a few seconds. At any 
+time, you can view all available OpenStack users by going to the 
+left-hand side pane on the {{extra.gui}} and selecting _Users_ > 
+_Openstack Users_.
+
+![All available OpenStack users](assets/ostack-cli/shot-06.png)
+
+## Creating and downloading an RC file
+
+On the {{extra.gui}} expand the left-hand side pane, click _Users_ and 
+then _Openstack Users_. You will then see, listed in the main pane 
+titled _Openstack Users_, all available users. Click on the 
+three-dotted round icon on the right of the user you want to create an 
+RC file for. From the pop-up that appears, select _Generate RC file_.
+
+![Generate  RC file](assets/ostack-cli/shot-07.png)
+
+The RC file will be generated automatically. Before downloading it onto 
+your local computer, you must select one of the available projects to 
+relate the RC file to. Do so and then click the blue _Download_ button.
+
+![Select project and download](assets/ostack-cli/shot-08.png)
+
+A "Save as" dialog window appears, specific to the operating system you 
+are currently using. Select a convenient location and save your RC file.
+ 
+## Modifying and sourcing the RC file 
+
+The general naming for RC files goes like this:
+
+```plain
+your_username--region_name--project_name--rc
+```
+
+So, assuming your username is `olafsdottir`, and the RC file has been 
+created for the `fra1` region and the `katla` project, your RC file 
+name should be this:
+
+```plain
+olafsdottir--fra1--katla--rc
+```
+
+Take a look at the contents of this file --- they should be like this:
+
+```bash
+export OS_USERNAME=olafsdottir
+export OS_PASSWORD=<your password goes here>
+export OS_AUTH_URL=https://fra1.citycloud.com:5000
+export OS_USER_DOMAIN_NAME=...
+export OS_PROJECT_DOMAIN_NAME=...
+export OS_REGION_NAME=Fra1
+export OS_PROJECT_NAME="katla"
+export OS_TENANT_NAME="katla"
+export OS_AUTH_VERSION=3
+export OS_IDENTITY_API_VERSION=3
+```
+
+Before you source the RC, and thus initialize all relevant environment 
```

Comment:
Sorry, missed this the first time around. Instead of just "RC", this should be "RC file".